### PR TITLE
Add CITATION.bib and update README.md

### LINKED ISSUE #50: CITATION.bib
What would be the best reference to cite this package? :)

### PR DESCRIPTION
**Pull request by abelsiqueira**
Closes #50

### CODE REVIEW

**Review by dpo — COMMENTED** [review]
LGTM but why is the year 2020 instead of 2021?

**Comment by abelsiqueira** [comment]
The Zenodo was created last year, we just never updated the README.

File: CITATION.bib [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@Misc{orban-siqueira-jsosolvers-2020,
+  author = {D. Orban and A. S. Siqueira and {contributors}},
+  title = {{JSOSolvers.jl}: {JuliaSmoothOptimizers} optimization solvers},
+  month = {August},
+  howpublished = {\url{https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl}},
+  year = {2020},
+  DOI = {10.5281/zenodo.3991143},
+}
```
Comment (matbesancon) [inline CITATION.bib:8]:
```suggestion
}

```
newline at the end of file?

**Comment by dpo** [comment]
It’s ok that the DOI was created last year. The DOI isn’t year specific. The citation should reflect the state of the repo when the bib was created.

File: CITATION.bib (hunk 1 above)
Comment (abelsiqueira) [reply CITATION.bib]: :+1:

**Comment by abelsiqueira** [comment]
Updated

File: README.md [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
 [![Cirrus CI - Base Branch Build Status](https://img.shields.io/cirrus/github/JuliaSmoothOptimizers/JSOSolvers.jl?logo=Cirrus%20CI)](https://cirrus-ci.com/github/JuliaSmoothOptimizers/JSOSolvers.jl)
-[![codecov](https://codecov.io/gh/JuliaSmoothOptimizers/JSOSolvers.jl/branch/master/graph/badge.svg?token=eyiGsilbZx)](https://codecov.io/gh/JuliaSmoothOptimizers/JSOSolvers.jl)
-[![](https://img.shields.io/badge/docs-latest-3f51b5.svg)](https://JuliaSmoothOptimizers.github.io/JSOSolvers.jl/latest)
 
 This package provides optimization solvers curated by the JuliaSmoothOptimizers
 organization.
+
+## How to cite
+
+If you use JSOSolvers.jl in your work, please cite using the format given in [CITATION.bib](CITATION.bib).
```
Comment (dpo) [inline README.md:17]: Does this link work? In other repos, we use a path to the master/main branch.

Comment (matbesancon) [reply README.md:17]: just checked, it does yes

**Comment by dpo** [comment]
Thank you.

**Pull request merged**